src/ceph.in

```diff
@@ -333,7 +333,7 @@ def parse_cmdargs(args=None, target=''):
                         help="make less verbose")
 
     parser.add_argument('-f', '--format', choices=['json', 'json-pretty',
-                        'xml', 'xml-pretty', 'plain'], dest='output_format')
+                        'xml', 'xml-pretty', 'plain', 'yaml'], dest='output_format')
```

Comment:
we should open a tracker item for a YAMLFormatter for common/Formatter.h

Reply:
https://tracker.ceph.com/issues/44745